src/pages/nft/claim/index.vue

```diff
@@ -890,6 +890,8 @@ export default {
       this.navigateToState(NFT_CLAIM_STATE.ERROR);
     } else if (state === NFT_CLAIM_STATE.LOGIN && this.claimingAddress) {
       this.navigateToState(NFT_CLAIM_STATE.ID_CONFIRMATION);
+    } else if (this.loginAddress) {
```

Comment:
Can we merge these 2 condition `state === NFT_CLAIM_STATE.LOGIN && this.claimingAddress` and  `this.loginAddress` ?